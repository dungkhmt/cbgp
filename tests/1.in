20 20
20 49
8 19
6 5
3 7
19 5
5 2
5 10
8 5
15 19
8 2
2 3
11 13
9 1
12 12
12 14
18 10
5 11
16 4
6 18
13 5
10 17
5 3
2 2
11 10
12 2
2 7
2 4
16 3
19 10
8 15
20 1
1 2
9 20
12 20
2 20
2 16
10 6
