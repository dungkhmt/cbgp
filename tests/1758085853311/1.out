(16, 12), (8, 15), (10, 6), (14, 18), (5, 1), (2, 1), (0, 13), (2, 0), (6, 5), (3, 2), (3, 5), (14, 11), (14, 0), (11, 3), (4, 15), (3, 8), (7, 5), (11, 19), (8, 19), (6, 2), (11, 8), (14, 3), (9, 13), (4, 0), (3, 18), (0, 11), (3, 6), (9, 19), (0, 1), (0, 8), (7, 17), (17, 5), (18, 11), (10, 3), (0, 15), (0, 16), (0, 5), (12, 1), (4, 8), (14, 2), (2, 7), (0, 9), (3, 0), (4, 3), (0, 19), 
numberIntersection=0 
minAngle=0.01801606855405513 
sumAngle=18.58589143443938 
minEdgeLength=1.0 
sumEdgeLength=473.0400204175678 
minDistance=0.16609095970747995 
sumDistance=6210.378262508234 

0.0000000000 0.0180160686 18.5858914344 1.0000000000 473.0400204176 0.1660909597 6210.3782625082
0: (36, 0),
1: (16, 16),
2: (0, 0),
3: (20, 10),
4: (21, 8),
5: (16, 15),
6: (17, 11),
7: (13, 12),
8: (19, 6),
9: (18, 3),
10: (18, 12),
11: (15, 2),
12: (17, 17),
13: (19, 4),
14: (12, 1),
15: (21, 7),
16: (19, 17),
17: (16, 12),
18: (14, 3),
19: (19, 5),
36 0 16 16 0 0 20 10 21 8 16 15 17 11 13 12 19 6 18 3 18 12 15 2 17 17 19 4 12 1 21 7 19 17 16 12 14 3 19 5 
0: (36, 0), 1: (16, 16), 2: (0, 0), 3: (20, 10), 4: (21, 8), 5: (16, 15), 6: (17, 11), 7: (13, 12), 8: (19, 6), 9: (18, 3), 10: (18, 12), 11: (15, 2), 12: (17, 17), 13: (19, 4), 14: (12, 1), 15: (21, 7), 16: (19, 17), 17: (16, 12), 18: (14, 3), 19: (19, 5), 
numberIntersection=0 minAngle=0.01801606855405513 sumAngle=18.58589143443938 minEdgeLength=1.0 sumEdgeLength=473.0400204175678 minDistance=0.16609095970747995 sumDistance=6210.378262508234 
Time taken: 719.652 s
